src/image/pixels.js

```diff
@@ -427,17 +427,17 @@ p5.prototype.filter = function(operation, value) {
  * Getting the color of a single pixel with get(x, y) is easy, but not as fast
  * as grabbing the data directly from <a href="#/p5/pixels">pixels[]</a>. The equivalent statement to
  * get(x, y) using <a href="#/p5/pixels">pixels[]</a> with pixel density d is
- * <code>
- * var x, y, d; // set these to the coordinates
- * var off = (y * width + x) * d * 4;
- * var components = [
+ * ```javascript
```

Comment:
I don't think this will render correctly. have you tested it?

Reply:
Yeah I tested it. It currently displays as code but is hard to read because it doesn't have any line breaks you can see that here:

![currentget](https://user-images.githubusercontent.com/43045568/50380822-f2e88680-0631-11e9-8c75-46d3ed4c8be1.jpg)

I chose to match the syntax that I saw in other places throughout the image docs. Here's the suggested change displayed with `npm run docs:dev`:

![newget](https://user-images.githubusercontent.com/43045568/50380823-f2e88680-0631-11e9-8cdc-12a186ea8ecc.jpg)

I can change it back if you like but I think this is easier to understand.

Reply:
this looks good. thanks for posting the screenshot.